tests/console/validate_raid_encrypt_home.pm

```diff
@@ -22,7 +23,7 @@ sub run {
 
     assert_equals($md_name, $lsblk_output->{blockdevices}[0]{name}, "Multi-disk name not found");
     assert_equals('crypt', $lsblk_output->{blockdevices}[0]{children}[0]{type}, "Encrypted type not found");
-    assert_equals('/home', $lsblk_output->{blockdevices}[0]{children}[0]{mountpoints}[0], "Encrypted mount point not found");
+    assert_equals('/home', is_sle('>15-SP3') ? $lsblk_output->{blockdevices}[0]{children}[0]{mountpoints}[0] : $lsblk_output->{blockdevices}[0]{children}[0]{mountpoint}, "Encrypted mount point not found");
```

Comment:
If you can prove that <= 15SP3 is the same json output then no problem, otherwise I would suggest to set specific value for 15SP3. Maybe later set another value for 12+.

Reply:
this is actually correct, I remember this changed long time ago:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13762/files
could you try to use that function `is_lsblk_able_to_display_mountpoints` instead of `is_sle` ?

Reply:
This method is suitable for such situation. :)